From mkondrin@hppi.troitsk.ru  Mon Nov 17 16:38:03 2008
From: M Kondrin <mkondrin@hppi.troitsk.ru>
Date: Fri, 14 Nov 2008 13:02:45 +0300
Subject: USB: ACE1001 patch for cp2101.c
To: greg@kroah.com
Cc: linux-usb@vger.kernel.org
Message-ID: <491D4CC5.6000905@hppi.troitsk.ru>

From: M Kondrin <mkondrin@hppi.troitsk.ru>

The patch which adds IDs for AKTAKOM USB->RS232 cable
(http://www.aktakom.ru/product/kio/ace-1001.htm) is attached.

From: M Kondrin <mkondrin@hppi.troitsk.ru>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/serial/cp2101.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/usb/serial/cp2101.c
+++ b/drivers/usb/serial/cp2101.c
@@ -56,6 +56,7 @@ static void cp2101_shutdown(struct usb_s
 static int debug;
 
 static struct usb_device_id id_table [] = {
+	{ USB_DEVICE(0x0471, 0x066A) }, /* AKTAKOM ACE-1001 cable */
 	{ USB_DEVICE(0x0489, 0xE000) }, /* Pirelli Broadband S.p.A, DP-L10 SIP/GSM Mobile */
 	{ USB_DEVICE(0x08e6, 0x5501) }, /* Gemalto Prox-PU/CU contactless smartcard reader */
 	{ USB_DEVICE(0x0FCF, 0x1003) }, /* Dynastream ANT development board */
